db/migrate/20210217160425_add_lesson_content_part.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddLessonContentPart < ActiveRecord::Migration[6.1]
+  def change
+    create_table :course_lesson_parts do |t|
+      t.timestamps
+
+      t.column :title, :string, null: false
+      t.column :content, :text, null: false, limit: 100_000
+
+      t.references :previous_lesson_part, null: true, foreign_key: { to_table: :course_lesson_parts }, type: :uuid
+      t.references :course_lesson, null: false, foreign_key: true, type: :uuid
```

Comment:
Do we want to add any indexes? Open question - just a thought

Reply:
I think foreign key gives us index for free.

My schema has
```
t.index ["course_lesson_id"], name: "index_course_lesson_parts_on_course_lesson_id"
t.index ["previous_lesson_part_id"], name: "index_course_lesson_parts_on_previous_lesson_part_id"
```
which are pretty much the things I want to index on.

Reply:
👍